migrations/8_token_acc_key.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE fungible_tokens ADD COLUMN fbt_pubkey bytea NOT NULL;
+
+ALTER TABLE fungible_tokens DROP CONSTRAINT fungible_tokens_pkey;
+
+ALTER TABLE fungible_tokens ADD PRIMARY KEY (fbt_pubkey);
+
+DROP INDEX IF EXISTS fungible_tokens_fbt_balance_idx;
+
+DROP INDEX IF EXISTS fungible_tokens_fbt_slot_updated_idx;
+
+CREATE INDEX fungible_tokens_fbt_owner_balance_idx ON fungible_tokens(fbt_owner, fbt_balance);
```

Comment:
Those indexes should be dropped and recreated in batch load in Postgres, can't see that in this pr